### ISSUE TITLE
Resize direction function doesn't work

### ISSUE THREAD

**Issue by wenpengy** [fvwmorg/fvwm3#881]
Thanks for reporting your bug here!  The following template will help with
giving as much information as possible so that it's easier to diagnose and
fix.

## Upfront Information

Please provide the following information by running the command and providing
the output.

* Fvwm3 version (run: `fvwm3 --version`)

fvwm3 1.0.7 (released)
with support for:  ReadLine, XPM, PNG, SVG, Shape, XShm, SM, XRandR, XRender, XCursor, XFT, NLS

fvwm3 comes with NO WARRANTY, to the extent permitted by law. You may
redistribute copies of fvwm under the terms of the GNU General Public License.
For more information about these matters, see the file named COPYING.

* Linux distribution or BSD name/version

OS: Gentoo Linux x86_64

* Platform (run: `uname -sp`)

Linux Intel(R) Core(TM) i7-4790K CPU @ 4.00GHz

## Expected Behaviour

What were you trying to do?  Please explain the problem.

I want to resize the window, but the coordinates of the upper left corner remain the same, so I add the "direction SouthEast" parameter after the resize command. 

Current resize direction SouthEast w+100p w+100p

## Actual Behaviour

The window resize w+100p w+100p.  However, the window extends from the center to the four sides , and the coordinates of the upper left corner is changed.

**Comment by ThomasAdam** [comment]
Hi @wenpengy 

In testing this using xterm, your command is working how I would expect it to -- as in, the NW corner of the window remains where it is, and the window grows in a SE direction.

For the window you're using, can you run the command `xprop` on it, and send me the output?  I'm specifically interested to know if the `WM_NORMAL_HINTS(WM_SIZE_HINTS)` XAtom contains a window gravity property.

For this specific window, it might be worth you looking at the Style options of::

* UseGravity
* IgnoreGravity

To see if that changes anything for you,

**Comment by wenpengy** [comment]
> 
Thank you!

Following your tips, I tested other program windows and they worked fine. The window program that doesn't work is kitty.  Here is the xprop output of kitty: WM_NORMAL_HINTS(WM_SIZE_HINTS): window gravity: Static.

I tested  style options of UseGravity and  IgnoreGravity. However, the window still extends from the center to the four sides.  The bug should be caused by the kitty program itself.

Thanks again.
